### PR TITLE
TPWDCEE-6329.

### PR DESCRIPTION
**Pull request by ozada**

### CODE REVIEW

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=5275545&bid=17276896)

File: view/frontend/web/js/view/payment/method-renderer/creditcard.js [hunk 1]
```diff
@@ -92,13 +92,17 @@ define(
...
             getData: function () {
-                return {
-                    "method": this.getCode(),
-                    "po_number": null,
-                    "additional_data": {
-                        "is_active_payment_token_enabler": this.vaultEnabler.isActivePaymentTokenEnabler()
+                var data =  {
+                    method: this.getCode(),
+                    po_number: null,
+                    additional_data: {
```
Comment (rinnhofer) [inline view/frontend/web/js/view/payment/method-renderer/creditcard.js]: Codacy Issue found: [Identifier 'additional_data' is not in camel case.](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=5275545)

File: view/frontend/web/js/view/payment/method-renderer/creditcard.js [hunk 2]
```diff
@@ -92,13 +92,17 @@ define(
...
+                        is_active_payment_token_enabler: false,
```
Comment (rinnhofer) [inline view/frontend/web/js/view/payment/method-renderer/creditcard.js]: Codacy Issue found: [Identifier 'is_active_payment_token_enabler' is not in camel case.](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=5275545)

File: view/frontend/web/js/view/payment/method-renderer/creditcard.js [hunk 3]
```diff
@@ -92,13 +92,17 @@ define(
...
              */
             getData: function () {
-                return {
-                    "method": this.getCode(),
-                    "po_number": null,
-                    "additional_data": {
-                        "is_active_payment_token_enabler": this.vaultEnabler.isActivePaymentTokenEnabler()
+                var data =  {
+                    method: this.getCode(),
+                    po_number: null,
```
Comment (rinnhofer) [inline view/frontend/web/js/view/payment/method-renderer/creditcard.js]: Codacy Issue found: [Identifier 'po_number' is not in camel case.](https://app.codacy.com/gh/wirecard/magento2-ee/pullRequest?prid=5275545)

File: view/frontend/web/js/view/payment/method-renderer/creditcard.js [hunk 4]
```diff
@@ -92,13 +92,17 @@ define(
              * Get the data
              */
```
Comment (edra-bumci) [inline view/frontend/web/js/view/payment/method-renderer/creditcard.js]: @ozada Missing return annotation, can you also add the return annotation to getFormId(thats my part :) )

**Pull request merged**